docs/content/latest/admin/cqlsh.md

```diff
@@ -16,11 +16,11 @@ showAsideToc: true
 
 ## Overview
 
-`cqlsh` is a command line shell for interacting with YugaByte DB through [YCQL](../../api/ycql/). 
+`cqlsh` is a command line shell for interacting with YugaByte DB through [YCQL](/latest/api/ycql/).
```

Comment:
Why are we replacing relative links with hard-coded links? This will make further branching harder.

Reply:
Fixed -- replaced with relative links.